simphony_ui/tests/test_cfd_dem.py

```diff
@@ -0,0 +1,25 @@
+"""
+Tests
+"""
+
+import unittest
+import simphony_ui.couple_CFDDEM as cfd_dem
+import shutil
+import os
+import tempfile
+
+
+class TestCfeDem(unittest.TestCase):
+
+    def setUp(self):
+        self.tmp_dir = tempfile.mkdtemp()
+        self.mesh_name = 'test_mesh'
+
+    def test_output(self):
+        cfd_dem.main(self.tmp_dir, self.mesh_name)
+        self.assertTrue(os.path.exists(
+            os.path.join(self.tmp_dir, self.mesh_name)
+        ))
+
+    def tearDown(self):
```

Comment:
it can happen that you never get to create the directory. if setUp fails. It's good practice to just wrap it in the appropriate try:except for OSError.